agent/config/flags.go

```diff
@@ -72,6 +72,7 @@ func AddFlags(fs *flag.FlagSet, f *Flags) {
 	add(&f.Config.DNSDomain, "domain", "Domain to use for DNS interface.")
 	add(&f.Config.EnableScriptChecks, "enable-script-checks", "Enables health check scripts.")
 	add(&f.Config.EncryptKey, "encrypt", "Provides the gossip encryption key.")
+	add(&f.Config.Ports.GRPC, "grpc-port", "Sets the gRPC API port to listen on (currently needed for Envoy xDS only).")
```

Comment:
Note: up on line 62, `-client` help text should probably list `gRPC` too. Will change that.